client/blocks/reader-recommended-sites/index.jsx

```diff
@@ -61,7 +61,7 @@ export class RecommendedSites extends React.PureComponent {
 			<div className="reader-recommended-sites">
 				<h2 className="reader-recommended-sites__header text-subtitle">
 					<Gridicon icon="thumbs-up" size={ 18 } />
-					{ this.props.translate( 'Recommended Sites' ) }
+					{ this.props.translate( 'Recommended sites' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.